website/content/en/preview/faq.md

```diff
@@ -134,6 +134,15 @@ Today, Karpenter will warn you if the number of instances in your Provisioner is
 Technically, Karpenter has a concept of an “offering” for each instance type, which is a combination of zone and capacity type (equivalent in the AWS cloud provider to an EC2 purchase option – Spot or On-Demand).
 Whenever the Fleet API returns an insufficient capacity error for Spot instances, those particular offerings are temporarily removed from consideration (across the entire provisioner) so that Karpenter can make forward progress with different options.
 
+## Scheduling
+
+### When using preferred topology spreads, Karpenter launches the correct number of nodes at first.  Why do they then sometimes get consolidated immediately? 
```

Comment:
This is true for preferred antiaffinity, right? Or potentially even preferred requirements if it splits to pods that could fit together when the first node comes online.

Reply:
Yeah, that's right. I'll update.